spec/monthday.html

```diff
@@ -325,17 +325,12 @@ <h1>ToPartialMonthDay ( _monthDay_ )</h1>
       <h1>RegulateMonthDay ( _month_, _day_, _disambiguation_ )</h1>
       <emu-alg>
         1. Assert: _disambiguation_ is one of `"constrain"`, `"balance"`, or `"reject"`.
-        1. If _disambiguation_ is `"constrain"`, then
-          1. <mark>TODO: define.</mark>
-        1. If _disambiguation_ is `"balance"`, then
-          1. Return ! BalanceMonthDay(_month_, _day_).
-        1. If _disambiguation_ is `"reject"`, then
-          1. If ! ValidateMonthDay(_month_, _day_) is *false*, then
-            1. Throw a *RangeError* exception.
-          1. Return the new Record {
-              [[Month]]: _month_,
-              [[Day]]: _day_
-            }.
+        1. Let _leapYear_ be 1972.
```

Comment:
Any reason for 1972 in particular?

Reply:
Any leap year will do; this one was close to 1970 which was used in the polyfill before (presumably based on the epoch).